tests/normalize_case.rs

```diff
@@ -0,0 +1,101 @@
+use jst::{convert_avro, convert_bigquery};
+use jst::{Context, ResolveMethod};
+use pretty_assertions::assert_eq;
+use serde_json::Value;
+
+fn test_data() -> Value {
+    serde_json::from_str(
+        r#"
+    {
+        "type": "object",
+        "properties": {
+            "test_snake_case": {"type": "boolean"},
+            "testCamelCase": {"type": "boolean"},
+            "TestPascalCase": {"type": "boolean"},
+            "TEST_SCREAMING_SNAKE_CASE": {"type": "boolean"}
+        },
+        "required": [
+            "test_snake_case",
+            "testCamelCase",
+            "TestPascalCase",
+            "TEST_SCREAMING_SNAKE_CASE"
+        ]
+    }
+    "#,
+    )
+    .unwrap()
+}
+
+#[test]
+fn test_bigquery_normalize_snake_casing() {
+    let context = Context {
+        normalize_case: true,
+        resolve_method: ResolveMethod::Panic,
+    };
+    let expected: Value = serde_json::from_str(
+        r#"
+        [
+            {
+                "mode": "REQUIRED",
```

Comment:
That the order in here differs from the definition above is just pure coincidence (because order in these definitions is irrelevant), right?

Reply:
This is no coincidence, field names are sorted so schemas can be diff'ed in a deterministic way.